modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/files_query.rb

```diff
@@ -166,12 +174,18 @@ def root(id)
                             permissions: %i[readable writeable])
           end
 
-          def uri_path_for(folder)
+          def children_uri_path_for(folder)
             return "/v1.0/drives/#{@storage.drive_id}/root/children" if folder.root?
```

Comment:
🟡 Can't you use `location_uri_path_for` here somehow? Looks a bit WET ;).

Reply:
I wouldn't do so, as it will still need an `if` (appending `:/children` or `/children`).

so I'd argue, that it is actually better readable having it WET.